src/main/kotlin/org/radarbase/push/integration/common/auth/Oauth1Signing.kt

```diff
@@ -1,114 +1,37 @@
 package org.radarbase.push.integration.common.auth
 
 import okhttp3.Request
-import okhttp3.RequestBody
-import okio.Buffer
-import okio.ByteString
 import java.io.IOException
-import java.net.URLEncoder
-import java.security.GeneralSecurityException
-import java.util.*
-import javax.crypto.Mac
-import javax.crypto.spec.SecretKeySpec
+import kotlin.collections.HashMap
 
 class Oauth1Signing(
-    val oauthKeys: OauthKeys,
-    val nonce: String = UUID.randomUUID().toString(),
-    val timestamp: Long = System.currentTimeMillis() / 1000L
+    val parameters: HashMap<String, String>
```

Comment:
This can be just a regular `Map`  or a `LinkedHashMap` if it should be order-preserving. If it should be sorted by key, you can also use `TreeMap`.